commit 87b794d2ff61c51488e99ab28c85d6aa1530f7fb
Author: Jeremy Huddleston <jeremyhu@apple.com>
Date:   Sat Dec 10 19:47:18 2011 -0800

    Revert "Remove the install-clang-c makefile target."
    
    This reverts commit a97f02c40c95933df4b38a9ca7033836ded7ea18.

diff --git a/Makefile b/Makefile
index 35fa00f..79fb043 100644
--- Makefile
+++ Makefile
@@ -76,6 +76,14 @@ ifeq ($(MAKECMDGOALS),install-clang)
   NO_INSTALL = 1
 endif
 
+ifeq ($(MAKECMDGOALS),install-clang-c)
+  DIRS := tools/clang/tools/driver tools/clang/lib/Headers \
+          tools/clang/tools/libclang tools/clang/tools/c-index-test \
+         tools/clang/include/clang-c
+  OPTIONAL_DIRS :=
+  NO_INSTALL = 1
+endif
+
 ifeq ($(MAKECMDGOALS),clang-only)
   DIRS := $(filter-out tools docs unittests, $(DIRS)) \
           tools/clang tools/lto
@@ -163,6 +171,7 @@ clang-only: all
 tools-only: all
 libs-only: all
 install-clang: install
+install-clang-c: install
 install-libs: install
 
 # If SHOW_DIAGNOSTICS is enabled, clear the diagnostics file first.
